app/config/specs/open-api3-1.6.x-console.json

```diff
@@ -9451,9 +9451,6 @@
                                             "python-3.11",
                                             "python-3.12",
                                             "python-ml-3.11",
-                                            "deno-1.21",
-                                            "deno-1.24",
-                                            "deno-1.35",
                                             "deno-1.40",
```

Comment:
Are those intentional? If not, lets upgrade php-runtimes library to latest. Or check if it was removed there.

Reply:
its latest version, looks like generating specs was skipped when the update happened 👍🏻